README.md

```diff
@@ -100,7 +100,7 @@ client.login('token');
 ## Links
 
 - [Website](https://discord.js.org/) ([source](https://github.com/discordjs/website))
-- [Documentation](https://discord.js.org/#/docs)
+- [Documentation](https://old.discordjs.dev/#/docs)
 - [Guide](https://discordjs.guide/) ([source](https://github.com/discordjs/guide))
   See also the [Update Guide](https://discordjs.guide/additional-info/changes-in-v13.html), including updated and removed items in the library.
```

Comment:
I just noticed this! These links should ideally use https://v13.discordjs.guide since this is the version 13 branch.

```suggestion
- [Guide](https://v13.discordjs.guide) ([source](https://github.com/discordjs/guide/tree/v13))
  See also the [Update Guide](https://v13.discordjs.guide/additional-info/changes-in-v13.html), including updated and removed items in the library.
```